pkg/controller/kieapp/deploy_ui.go

```diff
@@ -438,7 +438,7 @@ func getServiceAccount(namespace string) *corev1.ServiceAccount {
 			Namespace: namespace,
 			Labels:    labels,
 			Annotations: map[string]string{
-				"serviceaccounts.openshift.io/oauth-redirectreference.primary": string(annotation),
+				"serviceaccounts.openshift.io/internal-registry-pull-secret-ref:console-cr-form-dockercfg-q7bbpserviceaccounts.openshift.io/oauth-redirectreference.primary": string(annotation),
```

Comment:
I think it can't be hardcoded as the service account name seems to contain generated strings on it.
console-cr-form-dockercfg-**q7bbp**serviceaccounts.openshift.io

Reply:
ack, updated